cgr/cgr.py

```diff
@@ -50,6 +50,19 @@ async def update_ratings_chameleon(self, ctx, players, winner):
         for player in players:
             await self.update_ratings_helper(ctx, [player], r2, 0.4, "chameleon")
 
+    async def update_ratings_blackjack(self, ctx, players, winners):
+        losers = set(players) - set(winners)
+        if (len(losers) > 0):
+            loser_rating = await self.get_average_rating(losers, "blackjack")
+            r2 = math.pow(10, loser_rating/400)
+            for loser in losers:
+                await self.update_ratings_helper(ctx, [loser], r2, 0.4, "blackjack")
```

Comment:
Why do you loop through if you're passing in the same params every time, you could just pass in the list losers